ui/components/MeshConfigComponent.js

```diff
@@ -140,7 +140,7 @@ async function uploadK8sConfig() {
   const field = document.getElementById("k8sfile");
   if (field instanceof HTMLInputElement) {
     if (field.files.length < 1) return;
-
+    const name=field.files[0].name;
```

Comment:
```suggestion
    const name = field.files[0].name;
```

Reply:
@SAEb-ai Could you please make this small change? Other than that, all others LGTM 🎉

Reply:
Done